packages/quick_actions/quick_actions_ios/example/ios/RunnerTests/FLTQuickActionsPluginTests.m

```diff
@@ -0,0 +1,151 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import Flutter;
+@import quick_actions_ios;
+@import quick_actions_ios.Test;
+@import XCTest;
+#import <OCMock/OCMock.h>
+
+@interface FLTQuickActionsPluginTests : XCTestCase
+
+@end
+
+@implementation FLTQuickActionsPluginTests
+
+// A dummy `UIApplicationShortcutItem`.
+- (UIApplicationShortcutItem *)searchTheThingShortcutItem {
+  return [[UIApplicationShortcutItem alloc]
+           initWithType:@"SearchTheThing"
+         localizedTitle:@"Search the thing"
+      localizedSubtitle:nil
+                   icon:[UIApplicationShortcutIcon
+                            iconWithTemplateImageName:@"search_the_thing.png"]
+               userInfo:nil];
+}
+
+// A dummy raw shortcut item.
+- (NSDictionary<NSString *, NSString *> *)searchTheThingRawItem {
+  return @{
+    @"type" : @"SearchTheThing",
+    @"localizedTitle" : @"Search the thing",
+    @"icon" : @"search_the_thing.png",
+  };
+}
+
+- (void)testHandleMethodCall_setShortcutItems {
+  FlutterMethodCall *call =
+      [FlutterMethodCall methodCallWithMethodName:@"setShortcutItems"
+                                        arguments:@[ self.searchTheThingRawItem ]];
```

Comment:
searchTheThingRawItem is a method not a property, it is recommended to use the method syntax instead of dot notation.
see: https://google.github.io/styleguide/objcguide.html#objective-c-method-names
"Dot notation is used only with property names, not with method names."